docs/Sprint-Review/sprint-27-summary.md

```diff
@@ -0,0 +1,62 @@
+# Sprint 27 Summary
+**08/03/2021 - 08/17/2021 **
+
+### Summary Updates
+Finish work on Secret Key mitigation.
```

Comment:
Did these all happen? Should we say "finish_ed_" work on secret key migration, etc.

Reply:
There are 2 tickets still to complete for the epic, but I referenced this bc it was part of our goal. But we havent quite met that goal yet.

Reply:
Got it. As written the updates seem more like the goals than what was accomplished -- could you rephrase so that is reflective of what was accomplished? e.g. "Continued work on Secret Key Migration" or "closed all but 2 on secret key migration"? This section is helpful for reporting purposes

Reply:
Sure thing!